update.sh

```diff
@@ -0,0 +1,4 @@
+git merge upstream/master
+git pull
+git commit
+git push
```

Comment:
Thanks for the PR! Just one question: what do we need `update.sh` for? It's basically for updating the local repository, right?